devbuild.sh

```diff
@@ -265,6 +265,19 @@ if [ "${DEFAULT_BUILD}" = true ]; then
   BUILD_UPP="on"
 fi
 
+# Allow the use of RRFS model output files remapped into CONUS 3-km grid as ICS/LBCS,
+# before UFS_UTILS integrates use of native/full RRFS files 
+# Files could be retrieved from 
+# https://noaa-rrfs-pds.s3.amazonaws.com/rrfs_a/rrfs_a.{yyyymmdd}/{hh}/control/
+# in the format rrfs.t{hh}z.prslev.f{fcst_hr:03d}.conus_3km.grib2
+if [ "${BUILD_UFS_UTILS}" = "on" ]; then
+    os=$(uname) &&  SED=sed
+    test $os == Darwin && ( os=MacOSX && SED=gsed )
+    echo "SED is ${SED}"
+
+    CHGRES_CUBE=${SRW_DIR}/sorc/UFS_UTILS/sorc/chgres_cube.fd
+    ${SED} -i 's/"RAP","HRRR"/"RAP","HRRR","RRFS"/g' ${CHGRES_CUBE}/program_setup.F90
+fi
```

Comment:
This is not an appropriate fix to fortran codes.

Reply:
This is a temporary workaround to allow chgres_cube to build before a later tag is used that officially introduces RRFS capability. The current UFS_UTILS tag simply flag the "RRFS" option as not valid, and reports an error. Newer UFS_UTILS tags do not yet work with the SRW, and require more adaptation.

Reply:
@christinaholtNOAA - 
please see comments:
https://github.com/ufs-community/ufs-srweather-app/pull/1089#issuecomment-2145746876
https://github.com/ufs-community/ufs-srweather-app/pull/1089#issuecomment-2150389008

Reply:
Remove an extra empty line

Reply:
The appropriate workaround to needing code not yet committed to an authoritative repo is to keep the necessary code in a branch and point to that branch in the interim. Editing external code in the build process is a bad practice that sets a horrible precedent.

Reply:
Updating the UFS_UTILS hash to [1dac855](https://github.com/ufs-community/UFS_UTILS/commit/1dac85544dd4134de7cb899214c7db67bbe8c0b4) would bring in the necessary change to `chgres_cube's` `program_setup.F90`.  However, all WE2E tests that use RAP and HRRR physics will fail while moving to this version of UFS_UTILS, as it is after the fractional grid update.  It doesn't look like there has been any work on issue [#961](https://github.com/ufs-community/UFS_UTILS/issues/961) in UFS_UTILS, so this is still keeping the SRW App from moving forward with updated versions of UFS_UTILS.

Reply:
I totally understand that there is no way to get the UFS_UTILS hash updated right now. 

There are two ways to handle it -- a temporary branch in a fork that EPIC controls to hold this source code change, or holding off on this PR until that update can be made.

Editing source code in the build script is unacceptable.

Reply:
@christinaholtNOAA @MichaelLueken 

Another possible approach could be to solve the issue of chgres_cube flagging the RRFS option to be done on SRW scripts level, instead of changing the error condition on the UFS_UTILS level. 

My suggestion is to change the variable "external_model" from "RRFS" to "HRRR" in namelist fort.41 files before chgres_cube is called. These namelist files are stored in $EXPTDIR/[TEST_NAME]/[YYYYMMDDHH]/tmp_MAKE_ICS/ and ./tmp_MAKE_LBCS/ .
The changes are to be done in  exregional_make_ics.sh and  exregional_make_lbcs.sh, or in exregional_get_extrn_mdl_files.sh

That way the RRFS files interpolated to a regular grid still could be retrieved from the RRFS AWS location as needed, according to the external model type set for the experiment, but will be processed by chgres_cube similar to  those for HRRR. This should be functional as long as HRRR and RAP remain as options for the SRW initial and lateral boundary condition.  

Please let me know if this sounds a viable alternative to error flag removal in chgres_cube's program_setup.F90!

Reply:
Please remove lines 268 - 279.  These lines are no longer required to allow RRFS ICs/LBCs following your updates to `exregional_make_ics` and `exregional_make_lbcs`.